lighthouse-core/report/html/report-styles.css

```diff
@@ -993,9 +994,18 @@
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
   color: var(--report-text-color-secondary);
-  margin: var(--category-padding);
-  padding: var(--category-padding);
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
```

Comment:
50% seems a little arbitrary and has the box edges not matching up with the "grid" of the report.

```cs
max-width: calc(var(--report-width) - var(--category-padding) * 2);
```
this'll have it match most of the report content.